User Guide updates to reflect latest version of Data Browser (#5)
solid/userguide: views/source/userguide.md

---
description: User Guide For Source Data Browser view
---

# Source View User Guide

> The sources for data resources (e.g., .ttl, html, etc.) can be viewed and edited via the Source view. 

Reviewer: ```suggestion
The source of data resources (e.g., Turtle, HTML, etc.) can be viewed and edited via the Source view. 
```
